cds/modules/webhooks/tasks.py

```diff
@@ -62,38 +64,52 @@ def sse_publish_event(channel, type_, state, meta):
         current_sse.publish(data=data, type_=type_, channel=channel)
 
 
-def _factory_sse_task_base(type_=None):
+def _factory_avc_task_base(type_=None, undo_fun=None):
```

Comment:
Maybe instead of avc we could put deposit, i.e `_factory_deposit_task_base`. We will use some of the tasks in other submissions